src/main/generic/consensus/account/Address.js

```diff
@@ -30,6 +30,10 @@ class Address extends Primitive {
             && super.equals(o);
     }
 
+    toString() {

```

Comment:
As Pascal mentioned, we should stick to base64 internally for now. Since showing the hex representation of hashes/addresses is more of a user interface issue, the solution should be to just explicitly call `.toHex()` when displaying these objects.